src/Math-Tests-Complex/PMComplexNumberTest.class.st

```diff
@@ -637,6 +637,15 @@ PMComplexNumberTest >> testRaisedTo [
 	self assert: c3 imaginary closeTo: c imaginary
 ]
 
+{ #category : #'testing - mathematical functions' }
+PMComplexNumberTest >> testRaisedToFractionalPower [
+
+	| z expected |
+	z := 0 + 1 i.
```

Comment:
When $0 + 1 i = cos(\pi/2) + sin(\pi/2)i$, its cube root is $cos(\pi/6) + i sin(\pi/6) = \sqrt(3)/2 + 1/2i$ by De Moivre's theorem

Reply:
Looks good to me. For better clarification while the variable name z is commonly used for complex numbers, consider using a more descriptive name that conveys its purpose in the test. For example, you could use complexNumber instead.